radixdlt-core/radixdlt/src/main/java/com/radixdlt/tree/substate/BerkeleySubStateStore.java

```diff
@@ -159,18 +159,20 @@ public void process(
     PMTStorage refCounterPMTStorage = new RefCounterPMTStorage(cachedPMTStorage);
     var subStateTree = new SubStateTree(new PMT(refCounterPMTStorage, this.rootHash));
     List<Pair<SubstateId, byte[]>> values = new ArrayList<>();
-    for (REStateUpdate stateUpdate : txn.stateUpdates().toList()) {
-      values.add(Pair.of(stateUpdate.getId(), SubStateTree.getValue(stateUpdate.isBootUp())));
+    for (REStateUpdate stateUpdate : txn.stateUpdates().filter(REStateUpdate::isBootUp).toList()) {
```

Comment:
I think I'd prefer to see `txn.stateUpdates().filter(REStateUpdate::isBootUp).toList()` extracted into a properly named variable. This would simplify reading and provide clues to the reader without the need to visually parse the stream processing chain.

Also, I think using `var` would make code more convenient to read.

Reply:
done.